src/content/docs/2/guide/frontend/nuxt.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Nuxt
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Nuxt is a meta-framework for Vue.
+
+## Checklist
+
+- Use SSG by setting `ssr: false`. Tauri doesn't support server based solutions.
+- Set host to `0.0.0.0`.
+- Use `dist/` as `distDir` in `tauri.conf.json`.
+- Compile using `nuxi generate`.
+
+## Example Configuration
+
+<Tabs>
+	<TabItem label="npm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "npm run dev",
+		"beforeBuildCommand": "npm run generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="yarn">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "yarn dev",
+		"beforeBuildCommand": "yarn generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="pnpm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "pnpm dev",
+		"beforeBuildCommand": "pnpm generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+</Tabs>
+
+```ts
+export default defineNuxtConfig({
+	// (optional) Enable the Nuxt devtools
+	devtools: { enabled: true },
+	// Enable SSG
+	ssr: false,
+	// (optional) Disable telemetry
+	telemetry: false,
```

Comment:
Could fall under the category of best practises, telemetry should be opt-in and Tauri helps keep our users safe and protect their privacy

Reply:
I put a note under the checklist to cover this. I agree that it could be a point on safety and privacy, but I think that's a user-preference dependent on the company/project/person and not necessarily a best practice 🤔